Extension/Syncfusion-NuGet-Packages/NuGet-Packages.md

```diff
@@ -158,31 +158,31 @@ The following steps help you to configure and restore the Syncfusion NuGet packa
     N> If you need to Install v16.2.0.46 before Syncfusion NuGet packages, add the required Syncfusion platform NuGet feed links in the NuGet.config file. You can get the Syncfusion NuGet package feed link by clicking the **Copy URL** label from the required platform provided in the following link:
     <http://nuget.syncfusion.com>
      
-    ![Sample project configuration page in Visual Studio Online Application](NuGet-VisualStudioonline_images/NuGet-VisualStudioonline-img1.png)
+    ![Sample project configuration page in Visual Studio Online Application in NuGet Package Extension](nuget-packages_images/nuget-packages-extension-sample-project.png)
 
 2.  If your application doesn't have any build definitions, create a new build definition by clicking the **New Definition** button under **Build & Release**. 
 
-    ![Build and Release section in Visual Studio Online Application](NuGet-VisualStudioonline_images/NuGet-VisualStudioonline-img2.png)
+    ![Build and Release section in Visual Studio Online Application in NuGet Extension](nuget-packages_images/nuget-packages-extension-release-section.png)
 
-    ![Build process template selection section in Visual Studio Online Application](NuGet-VisualStudioonline_images/NuGet-VisualStudioonline-img3.png)
+    ![Build process template selection section in Visual Studio Online Application in NuGet Package Extension](nuget-packages_images/nuget-packages-extension-template-selection.png)
 
 ### NuGet installation
 
 1.  If your application has any build definitions, navigate to **Build & Release** tab and select the **Edit** option from corresponding application to include the NuGet restore process.
 
-    ![Build and release section with edit option in Visual Studio Online Application](NuGet-VisualStudioonline_images/NuGet-VisualStudioonline-img4.png)
+    ![Build and release section with edit option in Visual Studio Online Application in NuGet Packages Extension](nuget-packages_images/nuget-packages-extension-release-section.png)
```

Comment:
Recheck the image folder name.
Check and update in all places.

Reply:
Changed the image folder names